frontend/packages/dev-console/src/components/catalog/providers/useDevfileSamples.tsx

```diff
@@ -3,15 +3,16 @@ import { TFunction } from 'i18next';
 import { useTranslation } from 'react-i18next';
 import { CatalogItem, ExtensionHook } from '@console/dynamic-plugin-sdk';
 import { coFetchJSON } from '@console/internal/co-fetch';
-import { APIError } from '@console/shared';
+import { APIError, useActiveNamespace } from '@console/shared';
 import { DevfileSample } from '../../import/devfile/devfile-types';
 
 const normalizeDevfileSamples = (devfileSamples: DevfileSample[], t: TFunction): CatalogItem[] => {
   const normalizedDevfileSamples = devfileSamples.map((sample) => {
     const { name: uid, displayName, description, tags, git, icon, provider } = sample;
+    const [activeNamespace] = useActiveNamespace();
```

Comment:
This will not compile, `useActiveNamespace` is a hook, it can't be used in a function.

Reply:
Thanks, I have fixed the code